src/components/CartContainer/Cart.jsx

```diff
@@ -0,0 +1,88 @@
+import  { Component } from 'react';
+import PropTypes from 'prop-types';
+import ProductCard from '../ProductCardContainer/ProductCard';
+import { cartContainerStyles, inputStyles, buttonContainerStyles } from './styles';
+import Button from '../ButtonComponent/Button';
+
+class Cart extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      street: '',
+      house: ''
+    };
+  }
+
+  handleOrder = () => {
+    const { street, house } = this.state;
+    alert(`Order placed! Address: ${street}, ${house}`);
+  };
+
+  handleStreetChange = (event) => {
+    this.setState({ street: event.target.value });
+  };
+
+  handleHouseChange = (event) => {
+    this.setState({ house: event.target.value });
+  };
+
+  render() {
+    const { cartItems, removeFromCart } = this.props;
+    const { street, house } = this.state;
+
+    return (
+      <div style={cartContainerStyles}>
+        <h2 style={{ textAlign: 'center' }}>Shopping Cart</h2>
+        {cartItems.length === 0 ? (
+          <p>Your cart is empty</p>
+        ) : (
+          <div>
+            {cartItems.map((item, index) => (
+              <ProductCard
+                key={index}
+                product={item}
+                isInCart={true}
```

Comment:
In such case you can put just isInCart attribute without value. It will be true by default